app/src/main/res/layout/_navigation_header.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/43b06fc24f1ddcb819de2e20dacc4fe82bd270b7/app/src/main/res/layout/_navigation_header.xml#L2"></span>The resource <code>R.layout._navigation_header</code> appears to be unused</td>
    </tr>
  </tbody>
</table>